src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -372,4 +372,14 @@ public boolean isSectionChanged(StudentAttributes originalStudentAttribute) {
     public boolean isTeamChanged(StudentAttributes originalStudentAttribute) {
         return this.team != null && !this.team.equals(originalStudentAttribute.team);
     }
+    
+    /**
+     * Checks whether the edit form of student has changed the email value.
```

Comment:
Okay. Can I edit the doc comments for the two methods above (`isSectionChanged ` and `isTeamChanged`) in the same PR? They suffer from the same phrasing. Or must I open a new first-timer issue (how strict is our policy)?